### PR TITLE
Copy libs not present as symlinks in top lib directory

### PR DESCRIPTION
**Pull request by Noarkhh**

### CODE REVIEW

**Comment by bblaszkow06** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/membraneframework/elixir?version=dev:d1ae7786e6226b118f2da77d9b24c6784e622fd0

**Comment by bblaszkow06** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/membraneframework/elixir?version=dev:83d0d87737565be3e662cb02c244dcbde62a6df9

**Comment by bblaszkow06** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/membraneframework/elixir?version=dev:8be354239f380227294b29667e7efa3e0c5f3559

**Comment by bblaszkow06** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/membraneframework/elixir?version=dev:4857bc97250ce0d7b5ad453dbe0c386a9c07f7a0

**Comment by bblaszkow06** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/membraneframework/elixir?version=dev:844e0c7e5d1c27eef394555adab1c1f923d71736

File: src/scripts/precompile_for_macos.sh [hunk 1]
```diff
@@ -18,23 +18,24 @@ then
...
-        # not absolute, so that it can be located during linking
+        # not absolute (as brew sets it), so that it can be located during linking
         install_name_tool -id "@rpath/$(basename $f)" $f
-        # update LC_LOAD_DYLIB commands that refer to dependencies of current library that have been 
-        # installed by brew so that their names are also relative paths, so that the dependencies can be located during linking
-        otool -L $f | tail -n +3 | awk -F" " '{print $1}' | ( grep "^${brew_prefix}" || [ "$?" == "1" ] ) | while read -r line; do install_name_tool -change $line "@rpath/$(basename $line)" $f; done
+        # update the LC_LOAD_DYLIB commands that refer to dependencies of current library that have been 
+        # installed by brew and set as local paths so that their names are also relative paths and the dependencies can be located during linking
+        otool -L $f | tail -n +3 | awk -F" " '{print $1}' | ( grep -E "^(${brew_prefix}|@loader_path/../../../../opt)" || [ "$?" == "1" ] ) | while read -r line; do install_name_tool -change $line "@rpath/$(basename $line)" $f; done
```
Comment (mat-hek) [inline src/scripts/precompile_for_macos.sh:38]: Maybe grepping by `@loader_path` alone would be better? I imagine the number of `../` may change one day

**Review by mat-hek — APPROVED** [review]

File: src/scripts/precompile_for_macos.sh (hunk 1 above)
Comment (Noarkhh) [reply src/scripts/precompile_for_macos.sh:38]: I believe it's directly tied to the brew's file structure, but sure, better safe than sorry

**Comment by bblaszkow06** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/membraneframework/elixir?version=dev:00e05bbbf276d710514271e578b428057a9286b4

**Comment by bblaszkow06** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/membraneframework/elixir?version=dev:88b65dbe482b398c028ba51e04b9501175670025

**Pull request merged**